package.json

```diff
@@ -20,38 +20,46 @@
     "node": "^18.x || ^20.x "
   },
   "dependencies": {
-    "chalk": "^5.3.0",
+    "@aurodesignsystem/auro-button": "^8.0.0",
```

Comment:
each component has `package.json` to have their own dependencies. do we need these here?

Reply:
For this ticket it was a requirement to get the component's to individually function with their own respective requirements, etc. But as a monorepo, the `auro-form` component will need a tool like Lerna or Turbo to properly manage the components.

There is a ticket tracking that work here: https://github.com/AlaskaAirlines/auro-form/issues/19